templates/components/itilobject/service_levels.html.twig

```diff
@@ -31,215 +31,512 @@
+            {% set display_associatedsla_field = field_options.fields_template is not defined or not field_options.fields_template.isHiddenField('slas_id_ttr') %}
+            {% set can_assign_sla_ola = canupdate and has_profile_right('slm', constant('SLM::RIGHT_ASSIGN')) and not field_options.fields_template.isReadonlyField('slas_id_ttr') %}
+
+            <td class="right pe-2 align-middle border-0">{{ __('TTR') }}</td>
+            <td class="d-flex ms-n2 border-0 py-0">
+
+                {% set _rand = random() %}
+                {# state flag #}
+                <div class="align-middle my-2 me-2">
+                    <i class="ti ti-flag"></i>
+                </div>
+
+                {# datetime field #}
+                {% if display_datetime_field %}
+                    {{ _self.DatetimeLaField('time_to_resolve',
+                        item.fields['time_to_resolve'],
+                        __('TTR'),
+                        {
+                            'rand': _rand,
+                            'disabled': not can_assign_sla_ola
+                        }) }}
+                {% endif %}
+
+                {# no linked ttr : display add button + hidden dropdown #}
+                {% if display_associatedsla_field and can_assign_sla_ola %}
+                    {{ _self.LinkedLaFieldEmpty(ttr_sla,
+                        item,
+                        'slas_id_ttr',
+                        ttr_sla.getAddConfirmation(),
+                        constant('SLM::TTR'),
+                        options = {
+                            'can_assign': can_assign_sla_ola,
+                            'rand': _rand,
+                        }) }}
+                {% endif %}
+            </td>
+        </tr>
+    {% endif %}
+    </tbody>
+
+{# --------- OLA display ---------- #}
+    {% set display_associatedsla_field = field_options.fields_template is not defined or not field_options.fields_template.isHiddenField('_olas_id') %}
+    <thead class="{% if not display_associatedsla_field %}d-none{% endif %}">
+    <tr class="">
+        <th class="visually-hidden"></th>
+        <th class="border-0 fs-4">
+            <div class="d-flex justify-content-between">
+                <div class="">
+                    {% set ola_tto_is_mandatory = field_options.fields_template.isMandatoryField('_olas_id_tto') %}
+                    {% set ola_ttr_is_mandatory = field_options.fields_template.isMandatoryField('_olas_id_ttr') %}
+                    {% set required = ola_tto_is_mandatory or ola_ttr_is_mandatory %}
+                    {% set helper_text %}
+                        {% if ola_tto_is_mandatory %} {{ __('OLA') ~ ' ' ~ __('TTO') ~ ' ' ~ __('is mandatory') }} <br>{% endif %}
+                        {% if ola_ttr_is_mandatory %} {{ __('OLA') ~ ' ' ~ __('TTR') ~ ' ' ~ __('is mandatory') }} {% endif %}
```

Comment:
This will likely be problematic in a few other languages where word order needs to change. These will be better as:
`__('OLA TTO is mandatory')`
OR:
`sprintf(__('%s is mandatory'), __('OLA TOO'))`